driver/command/command_nix_test.go

```diff
@@ -0,0 +1,54 @@
+// +build !windows
+
+package command
+
+import (
+	"github.com/deislabs/cnab-go/bundle"
+	"github.com/deislabs/cnab-go/driver"
+	"github.com/stretchr/testify/assert"
+
+	"os"
+	"testing"
+)
+
+func TestCommandDriverOutputs(t *testing.T) {
+	content := `#!/bin/sh
+		mkdir -p "${CNAB_OUTPUT_DIR}//cnab/app/outputs"
```

Comment:
nit: I don't think you don't need the double `//` after CNAB_OUTPUT_DIR, just 1 slash.